utils/hash_containers.h

```diff
@@ -37,14 +37,33 @@ typedef struct hash_set_entry_s {
 } hash_set_entry_t;
 typedef hash_set_entry_t *hash_set_t;
 
+// TODO - map
+typedef struct cw_entry_t {
+  flex_trit_t hash[FLEX_TRIT_SIZE_243];
+  int64_t cw;
+  UT_hash_handle hh;
+} cw_entry_t;
+
+typedef cw_entry_t *cw_map_t;
+
+typedef struct hash_to_direct_approvers_entry_t {
+  flex_trit_t hash[FLEX_TRIT_SIZE_243]; /*key*/
+  uint64_t idx;
+  hash_set_t approvers; /*value*/
+  UT_hash_handle hh;
+} hash_to_direct_approvers_entry_t;
+
+typedef hash_to_direct_approvers_entry_t *hash_to_direct_approvers_map_t;
```

Comment:
I feel like these structs are not general purpose hash containers. Shouldn't they be in `consensus/model.h` instead ?

Reply:
yeah, let's refactor that later